Automattic/jetpack
# Reorg: Move debug-helper to a plugin

The `debug-helper` is in the `packages` dir right now, but it isn't really a composer package -- it is a plugin used for development purposes.

This would move it as a plugin.

#### Changes proposed in this Pull Request:
* moves debug to a plugin
* updates Docker to remove the special case for it (all plugin folders are added already).
* Adds composer.json file to prepare for the changes made in #18313.

#### Jetpack product discussion
- n/a
- 
#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Run `yarn docker:build-image && yarn docker:up`.
* Verify the debug plugin is a plugin on the install.

#### Proposed changelog entry for your changes:


*
